src/browserServerImpl.ts

```diff
@@ -52,9 +52,13 @@ export class BrowserServerLauncherImpl implements BrowserServerLauncher {
       env: options.env ? envObjectToArray(options.env) : undefined,
     }, toProtocolLogger(options.logger));
 
+    let path = `/${createGuid()}`;
+    if (options.path)
+      path = options.path.startsWith('/') ? options.path : `/${options.path}`;
```

Comment:
The leading slash is important. I opted to accept liberal input and fix it up. Let me know if we'd rather just throw an error if it doesn't start with `/`.

Likewise, do we want to do more validation or restriction on the path? Currently, you could put `/foo   bar` and we won't throw a validation error.

Reply:
You code looks fine as is.